<launch>

    <!-- rowmaster_combined -->
	<node name="navigation_task1" pkg="farmbeast_navigation" type="navigation_koncna" 		output="screen" > 
	<param name="obrni" value="-1" />
	<param name="min_val" value="0.02" />
	<param name="max_val" value="0.7" />
	<param name="max_obr" value="1" />
	<param name="xSmer" value="0.2" />
	<param name="st_mer" value="40"/>
	<param name="hitrost" value="0.1"/>
	</node>

   <!-- ovinek iz ene vrste v drugo -->
	


</launch>
